tasks/omnibus.py

```diff
@@ -265,9 +266,13 @@ def build(
                 with timed(quiet=True) as restore_cache:
                     # Allow failure in case the cache was evicted
                     if ctx.run(f"{aws_cmd} s3 cp --only-show-errors {git_cache_url} {bundle_path}", warn=True):
-                        print(f'Successfully restored cache {cache_key}')
-                        ctx.run(f"git clone --mirror {bundle_path} {omnibus_cache_dir}", err_stream=sys.stdout)
-                        cache_state = ctx.run(f"git -C {omnibus_cache_dir} tag -l").stdout
+                        print(f'Successfully retrieved cache {cache_key}')
+                        try:
+                            ctx.run(f"git clone --mirror {bundle_path} {omnibus_cache_dir}")
+                        except UnexpectedExit as exc:
+                            print(f"An error occurring while cloning the cache repo: {exc}")
```

Comment:
❓ Question :
Do we want to continue ? if it fails to clone shouldn't we either retry or exit ?

Reply:
We continue ignoring the cache (as if it had missed the cache). It looks like the most straightforward thing to do to ensure this non-fatal error doesn't break builds, no?

Reply:
If we do continue might good to add another metric to send to track clone UnexpectedExit

Reply:
Requiring would involve repeating the s3 download.

I agree that it may be useful, but I believe once we get around to improving the build order (what https://github.com/DataDog/datadog-agent/pull/25325 is trying to do) we should drastically reduce the number of uploads to the point that hitting this should be rare (and not very consequential).

Reply:
Tbh I'd still prefer to have the metric of it happening just in case but it can be done in a separate PR 👍